src/daemons/MetaDaemon.cpp

```diff
@@ -56,9 +60,16 @@ DEFINE_bool(daemonize, true, "Whether run as a daemon process");
 
 static std::unique_ptr<apache::thrift::ThriftServer> gServer;
 static std::unique_ptr<nebula::kvstore::KVStore> gKVStore;
+#if defined(__x86_64__)
+static std::unique_ptr<google_breakpad::ExceptionHandler> gExceptionHandler;
```

Comment:
Define the static variable in `SetupBreakpad.cpp`

Reply:
Emmm, i thought the global one should be declared in the same scope, anyway i'm ok with your idea.